src/meta/meta_bulk_load_service.cpp

```diff
@@ -68,7 +74,8 @@ void bulk_load_service::on_start_bulk_load(start_bulk_load_rpc rpc)
     auto &response = rpc.response();
     response.err = ERR_OK;
 
-    if (!_meta_svc->try_lock_meta_op_status(meta_op_status::BULKLOAD)) {
+    if (!FLAGS_enable_concurrent_bulk_load &&
+        !_meta_svc->try_lock_meta_op_status(meta_op_status::BULKLOAD)) {
```

Comment:
Bulkload process will be continued when meta failover, and it will lock meta_op_status too in func: do_continue_app_bulk_load. Add this check?

Reply:
Done